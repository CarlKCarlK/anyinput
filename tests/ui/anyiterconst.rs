use anyinput::anyinput;

#[anyinput]
pub fn any_str_len(s: AnyIter<3>) -> Result<usize, anyhow::Error> {
    let len = s.len();
    Ok(len)
}

fn main() {}
