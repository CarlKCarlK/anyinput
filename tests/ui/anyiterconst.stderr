error: Expected generic parameter to be a type.
 --> tests/ui/anyiterconst.rs:4:23
  |
4 | pub fn any_str_len(s: AnyIter<3>) -> Result<usize, anyhow::Error> {
  |                       ^^^^^^^^^^
